tf2_geometry_msgs/include/tf2_geometry_msgs/tf2_geometry_msgs.hpp

```diff
@@ -117,24 +134,6 @@ template <>
 inline
   std::string getFrameId(const geometry_msgs::msg::Vector3Stamped& t) {return t.header.frame_id;}
 
-/** \brief Apply a geometry_msgs TransformStamped to an geometry_msgs Vector type.
- * This function is a specialization of the doTransform template defined in tf2/convert.h.
- * \param t_in The vector to transform, as a timestamped Vector3 message.
- * \param t_out The transformed vector, as a timestamped Vector3 message.
- * \param transform The timestamped transform to apply, as a TransformStamped message.
- */
-template <>
-inline
-  void doTransform(const geometry_msgs::msg::Vector3Stamped& t_in, geometry_msgs::msg::Vector3Stamped& t_out, const geometry_msgs::msg::TransformStamped& transform)
-  {
-    KDL::Vector v_out = gmTransformToKDL(transform).M * KDL::Vector(t_in.vector.x, t_in.vector.y, t_in.vector.z);
-    t_out.vector.x = v_out[0];
-    t_out.vector.y = v_out[1];
-    t_out.vector.z = v_out[2];
-    t_out.header.stamp = transform.header.stamp;
-    t_out.header.frame_id = transform.header.frame_id;
-  }
-
```

Comment:
Why was this function moved?

Reply:
Same question applies to other functions.

Reply:
I moved to keep everything consistent. Firstly moved all msgs without time stamp before those with stamp for instance Pose was at the very end of the file while PoseStamped is in the middle of the file, also moved getCovarianceMatrix to PoseWithCovariance comment section because it's not using stamp anywhere.